### ISSUE TITLE
Replace ffmpeg with sharp for image resizing

### ISSUE THREAD

**Issue by taylortom** [adapt-security/adapt-authoring#377]
See `FileProcessor#createImageThumb`

We could replace sharp by ffmpeg to resize images, but since sharp is shipped with node, so there is no need eventually.

_taken from TODO comment_

**Comment by fkcailiniyazi** [comment]
Personally I would like to keep sharp, since t is shipped with node.

**Comment by fkcailiniyazi** [comment]
I've noticed that sharp is already used for image resizing, but FFmpeg-fluent is used for retrieving metadata from media files and also for generating video thumbnail. 

For metadata, I've tried other packages, which are not depending on FFmpeg, but they are not compatible with the MongoDB driver, and for video thumbnail, I couldn't find anything suitable for our needs. 

To sum up, we do use sharp for image resizing, but we still depend on FFmpeg for some other works to be done.
